resources/dicts/patrols/general/medcat.json

```diff
@@ -3571,5 +3571,250 @@
                 ]
             }
         ]
+    },
+    {
+        "patrol_id": "gen_med_app_mentor_advice",
+        "biome": ["any"],
+        "season": ["any"],
+        "types": ["herb_gathering"],
+        "tags": [],
+        "patrol_art": "gen_med_and_app",
+        "min_cats": 2,
+        "max_cats": 2,
+        "min_max_status": {
+            "medicine cat apprentice": [1, 1],
+            "medicine cat": [1, 1]
+        },
+        "weight": 20,
+        "intro_text": "While out collecting herbs, p_l notices how quiet and dejected app1 is acting. After p_l asks {PRONOUN/app1/object} what's wrong, app1 admits {PRONOUN/app1/subject} {VERB/app1/don't/doesn't} feel cut out to be a medicine cat.",
+	"decline_text": "Unsure of what to say, p_l continues the herb patrol as usual, leaving app1 to {PRONOUN/app1/poss} sour mood.",
+        "chance_of_success": 60,
+        "success_outcomes": [
+            {
+                "text": "Smiling softly, s_c reasures app1 that {PRONOUN/s_c/subject} didn't feel ready to be a medicine cat, either, when {PRONOUN/s_c/subject} {VERB/s_c/were/was} an apprentice. s_c tells app1 a story from {PRONOUN/s_c/poss} own apprenticeship, when {PRONOUN/s_c/subject} accidentally swapped all the thyme for yarrow. Great StarClan was {PRONOUN/s_c/poss} mentor unhappy when all their patients started throwing up! Before long, app1 is laughing at s_c's own mishaps, {PRONOUN/app1/poss} own troubles forgotten.",
+                "exp": 20,
+                "weight": 20,
+		"stat_trait": ["childish", "bold", "oblivious", "playful", "shameless", "troublesome", "adventurous", "rebellious"],
+		"can_have_stat": ["p_l"],
+                "herbs": ["random_herbs"],
+                "relationships": [
```

Comment:
Could have these successes also reduce dislike and jealousy.  I think it would also be appropriate to raise those positive values by a bit more, maybe 10 instead of 5? since this feels like a fairly important moment in the app's bonding with their mentor.